packages/create-react-native-library/src/utils/generateRNApp.ts

```diff
@@ -0,0 +1,127 @@
+import fs from 'fs';
```

Comment:
Maybe we can call it  `generateExampleApp`.
WDYT?

Reply:
<img width="663" alt="Screen Shot 2022-09-06 at 17 55 11" src="https://user-images.githubusercontent.com/23079646/188667767-c538f7ac-4dad-4da5-987a-748045332bd2.png">

It was like this then got changed

Reply:
`exampleAppGenerators` is different 😅

I think `generateReactNativeExample` or something like that is good, we'd also need `generateExpoExample`